spec/rubocop/cop/security/json_load_spec.rb

```diff
@@ -0,0 +1,31 @@
+# encoding: utf-8
+# frozen_string_literal: true
+
+require 'spec_helper'
+
+describe RuboCop::Cop::Security::JSONLoad, :config do
+  subject(:cop) { described_class.new(config) }
+
+  it 'accepts JSON.parse' do
+    inspect_source(cop, 'JSON.parse("{}")')
```

Comment:
You should probably add a test verifying that something like `MyModule::JSON.parse` doesn't generate an offence.